README.md

```diff
@@ -44,10 +44,12 @@ is the recommended way of maintaining a copy of the rules inside the
 changes also locally, test them and then push them back to the shared
 repository at a later time.
 
+We no longer care about importing the full commit history, so `--squash` should be used.
```

Comment:
We should still add those commits being squashed in the PR description, right?

Reply:
That would be nice, but cannot be done easily unless developers start using the script instead of following these manual instructions.

Reply:
Volunteers are welcome for working on integration of that script into this repo... see https://github.com/kubernetes-csi/csi-release-tools/issues/7

Reply:
Can you add a line here to ask developers to copy the automatically generated commit message (which lists all the commits) to the PR description?

Reply:
We need to be careful that these individual commit messages are not of the format "Merge pull request". It seems to be messing up the release notes tool and causes the same PR numbers from the sidecar repos to be picked up. https://github.com/kubernetes-csi/csi-driver-host-path/pull/238#discussion_r565535893

Reply:
@msau42 I think the csi-driver-host-path 1.5.0 release note generation got confused because there were still some `release-tools` updates in the history since 1.4.0 which were done without `--squash`, for example: https://github.com/kubernetes-csi/csi-driver-host-path/commit/e53f3e85b7819546d2b66f32d5d7d4e6dc1020bc

Or do you have an example where that is still a problem when using squashing exclusively?

Reply:
> Can you add a line here to ask developers to copy the automatically generated commit message (which lists all the commits) to the PR description?

I've extended the instructions, please take another look.

Reply:
I don't have an example, but it's something we should watch out for when doing future release notes. I'm not sure how strict the release-notes parsing is.

Reply:
It didn't cause problem for me when I generated release notes.  We probably can't wrap them with \`\`\`.